src/frontend-samples/swiping-viewport/SwipingComparisonApp.tsx

```diff
@@ -0,0 +1,301 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+
+import {
+  ClipPlane,
+  ClipPrimitive,
+  ClipVector,
+  ConvexClipPlaneSet,
+  Point3d,
+  Transform,
+  Vector3d,
+} from "@bentley/geometry-core";
+import {
+  ContextRealityModelProps,
+  FeatureAppearance,
+  Frustum,
+  RenderMode,
+  ViewFlagOverrides,
+} from "@bentley/imodeljs-common";
+import {
+  EditManipulator,
+  FeatureSymbology,
+  findAvailableUnattachedRealityModels,
+  GraphicBranch,
+  IModelApp,
+  IModelConnection,
+  RenderClipVolume,
+  SceneContext,
+  ScreenViewport,
+  TiledGraphicsProvider,
+  TileTreeReference,
+  Viewport,
+} from "@bentley/imodeljs-frontend";
```

Comment:
I think we should collapse each of these import statements into a single line.  At present, the imports take up an entire screen in the code editor at runtime.